src/Controller/BaseController.php

```diff
@@ -29,6 +30,8 @@ public function __construct(RequestStack $requestStack)
         $this->dispatcher = $GLOBALS['dispatcher'];
 
         $this->config = $GLOBALS['config'];
+
+        $this->logger = $logger;
```

Comment:
Also, you have to declare the property logger at the begining of the class:

private $logger;

Like for entityManager